### PR TITLE
Update phcovid_plot module

### PR DESCRIPTION
**Pull request by andrewnyu**
Hi Enzo,

Sending pull request with the ff. changes:
1. Update phcovid_plot module to fix formatting of x-axis
2. Update get_cases() to store date columns as datetime format
3. Update Sample network analysis.ipynb to reflect module updates. 

Regards,
Andrew

### CODE REVIEW

File: phcovid/phcovid.py [hunk 1]
```diff
@@ -70,6 +76,8 @@ def get_cases(
     df_aliased[["contacts", "num_contacts"]] = extract_contact_info(
         df_aliased.travel_history
     )
+    df_aliased["confirmation_date"] = df_aliased["confirmation_date"].apply(lambda x: fix_dates(x))
```
Comment (enzoampil) [inline phcovid/phcovid.py]: Can you generalize this by applying it to all the "date columns" via for loop? Can define date columns as list, then loop through this.

Something like

```
DATE_COLS = ["contacts", "num_contacts"]
...
# in get_cases
for col in DATE_COLS:
    df_aliased[col] = df_aliased[col].apply(lambda x: fix_dates(x))

```

File: phcovid/phcovid_plot.py [hunk 2]
```diff
@@ -14,46 +15,35 @@ def get_case_plot(df, date_col="confirmation_date"):
...
-            return np.nan
-
     case_dates = df[date_col]
-    case_dates = case_dates.apply(lambda x: get_date_part(x))
     case_count = pd.DataFrame(case_dates[~case_dates.isnull()].unique())
     case_count.rename(columns={0: "date"}, inplace=True)
+    # sort values on date
+    case_count = case_count.sort_values("date", ascending = True)
+
+    if (start_date!=None):
```
Comment (enzoampil) [inline phcovid/phcovid_plot.py]: Can change to:

```
if start_date:
```

**Comment by enzoampil** [comment]
Hi @andrewnyu , Thanks for this! I left some comments :) 

Also, can you make sure to apply auto code formatting with `black` so our formats stay consistent?

In case you're not yet familiar just do the steps below in your command line / terminal:

```
# Install black
pip install black

# Format each python file you've changed
# In this case, phcovid_plot.py and phcovid.py
black phcovid_plot.py
black phcovid.py

# Commit and push your changes :) 
```

**Comment by andrewnyu** [comment]
Hi Enzo,

Got this. Updated pull request with discussed code and formatting changes

**Pull request merged**